### ISSUE TITLE
Question on the ASP.NET MVC Route debugger

### ISSUE THREAD

**Issue by argoc** [Haacked/feedback#148]
Hi, I know you've moved on to Git (congrats on the job change) but my students in CIS 229 have been trying out the Route Debugger you wrote for MVC, and are finding it's not reporting on attribute routing "correctly" -- it's saying false when the URL visually appears to match the route listed. The source code you provide in your old blog post is older than the current NuGet package, so getting into that to fix it or determine the cause seems like it won't be useful. Do you have a pointer to the current source you could point me at, or some other advice on how I can help them noodle their way through route debugging? (these are AAS-T students at Olympic College in Bremerton, the course teaches them ASP.NET MVC 5 using the book Professional ASP.NET MVC 5, and Chapter 11 refers to the Route Debugger).
Thank you,
Amelia
ps. sorry if any of this is hard to follow -- I've come down with the flu for the long weekend. lucky me.

**Comment by haacked** [comment]
Hi! The source code for the route debugger is in the https://github.com/haacked/routemagic repository. That repository contains a solution with multiple routing related projects. The [RouteDebug](https://github.com/Haacked/RouteMagic/tree/master/src/RouteDebug) project is the one you want.

I accept contributions! So if you and your students can fix it for ASP.NET MVC 5, I'm happy to cut a new release with your improvements.

**Comment by argoc** [comment]
Thank you - we'll take a look at it.
